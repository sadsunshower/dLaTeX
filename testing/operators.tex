\documentclass{article}

\begin{document}

Test: operators

It should catch these: $sin(x)$

\[
    sin(x)
\]

\[
cos(x)
\]

\[tan(x)\]

\[
    sin
\]

But not these: $\sin$

\[
    \cos
\]

\[
\tan    
\]

\[\cot\]

These shouldn't be caught, but are: sine

sin

orangutan

\end{document}