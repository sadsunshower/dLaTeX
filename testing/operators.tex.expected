This is dLaTeX version 0.1
{"messages": [{"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "It should catch these: $sin(x)$", "num": 6, "marker": [24]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "    sin(x)", "num": 9, "marker": [4]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "cos(x)", "num": 13, "marker": [1]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "\\[tan(x)\\]", "num": 16, "marker": [2]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "    sin", "num": 19, "marker": [4]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "These shouldn't be caught, but are: sine", "num": 34, "marker": [36]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "sin", "num": 36, "marker": [1]}], "error": false}
